_includes/comment.html

```diff
@@ -1,6 +1,6 @@
 <article id="comment{{ include.index }}" class="js-comment comment" itemprop="comment" itemscope itemtype="http://schema.org/Comment">
   <div class="comment__avatar-wrapper">
-    <img class="comment__avatar" src="https://www.gravatar.com/avatar/{{ include.email }}?d=mm&s=80">
+    <img class="comment__avatar" src="https://www.gravatar.com/avatar/{{ include.email }}?d=mm&s=80" alt="Avatar for {{ include.name }}">
```

Comment:
I removed the Avatar part which seems to be more clever just to have the name instead of having to localize. I could do what you originally suggested also, if you think that it is more clever.